app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -29,6 +30,68 @@ def save_container_projects_inventory(ems, hashes, target = nil)
     store_ids_for_new_records(ems.container_projects, hashes, :ems_ref)
   end
 
+  def save_container_quotas_inventory(ems, hashes, target = nil)
+    return if hash.nil?
+    target = ems if target.nil?
+
+    ems.container_quotas(true)
+    deletes = if target.kind_of?(ExtManagementSystem)
+                ems.container_quotas.dup
+              else
+                []
+              end
+
+    hashes.each do |h|
+      h[:container_project_id] = h.fetch_path(:project, :id)
+      h[:project] = nil
```

Comment:
no need for that, since it is in the list of ignored_keys in save_inventory_multi
removed, verified